### PR TITLE
Add ability to click on icome nad expense columns on balance graph

### PR DESCRIPTION
**Pull request by dissident**

### CODE REVIEW

**Comment by dissident** [comment]
https://trello.com/c/Y0n0auYB/331-balance

File: app/assets/javascripts/statistics.js [hunk 1]
```diff
@@ -79,7 +79,7 @@ var drawChart = function drawChart(period, element) {
       if (selectedItem) {
         var item = pieData.getValue(selectedItem.row, 0);
         if (item.indexOf('Other') == -1) {
-          for (var i = data.length - 1; i >= 0; i--) {
+          for (i = data.length - 1; i >= 0; i--) {
```
Comment (shir) [inline app/assets/javascripts/statistics.js]: @dissident why did you remove `var`?

Comment (dissident) [inline app/assets/javascripts/statistics.js]: Linter told me to do what, I can return var back.

Comment (shir) [inline app/assets/javascripts/statistics.js]: @dissident yes, return it please
